.ci/jobs.t/elastic+elasticsearch+dra-staging.yml

```diff
@@ -0,0 +1,71 @@
+---
+- job:
+      name: elastic+elasticsearch+%BRANCH%+dra-staging
+      workspace: /dev/shm/elastic+elasticsearch+%BRANCH%+dra-staging
+      display-name: "elastic / elasticsearch # %BRANCH% - DRA staging"
+      description: "Publishing Daily Releasable Artifacts (DRAs) of Elasticsearch %BRANCH% staging.\n"
+      node: "ubuntu-20.04"
+      builders:
+          - inject:
+                properties-file: '.ci/java-versions.properties'
+                properties-content: |
+                    JAVA_HOME=$HOME/.java/$ES_BUILD_JAVA
+                    RUNTIME_JAVA_HOME=$HOME/.java/$ES_RUNTIME_JAVA
+          - shell: |
+                #!/usr/local/bin/runbld --redirect-stderr
+                WORKFLOW="STAGING"
+                export WORKFLOW
+                ES_VERSION=$(cat build-tools-internal/version.properties \
+                      | grep elasticsearch \
+                      | sed "s/elasticsearch//g" \
+                      | sed "s/ //g" \
+                      | sed "s/=//")
+                VERSION_SUFFIX="" && [[ "$WORKFLOW" == "SNAPSHOT" ]] && VERSION_SUFFIX="-SNAPSHOT"
+                BEATS_BUILD_ID="$(./.ci/scripts/resolve-dra-manifest.sh beats master)"
```

Comment:
I'd prefer we pass $WORKFLOW as an explicit param here rather than rely on the implicit lookup of the environment variable.

Reply:
fixed